pilot/pkg/serviceregistry/serviceentry/controller_test.go

```diff
@@ -1959,13 +1961,20 @@ func Test_autoAllocateIP_values(t *testing.T) {
 
 	gotIPMap := make(map[string]string)
 	for _, svc := range gotServices {
-		if svc.AutoAllocatedIPv4Address == "" || doNotWant[svc.AutoAllocatedIPv4Address] {
+		if svc.AutoAllocatedIPv4Address == "" {
```

Comment:
this isn't an error scenario?

Reply:
Yes. It is . I am still looking at test case